src/Application/index.js

```diff
@@ -52,7 +52,7 @@ export default function(App, appData, platformSettings) {
     _setup() {
       Promise.all([
         this.loadFonts((App.config && App.config.fonts) || (App.getFonts && App.getFonts()) || []),
-        Locale.load((App.config && App.config.locale) || (App.getLocale && App.getLocale())),
+        initLocale(),
```

Comment:
This means that the Locale plugin is _always_ initialized (and tries to read locale files - that might not exist), even if the App doesn't implement  Localization

Reply:
`static get locale(){
    return true
}`

Needs to be added to App.js to init locale

Reply:
Update:

`static get locale(){ return true } ` Will load browser default language.
`static get locale(){ return false }` Will not load `Locale` on init.
`static get locale(){ return 'en-CA' }` Will load preferred language on init ('en-CA' for example)